UIKit/ML App UIKit/ViewController.swift

```diff
@@ -23,6 +23,7 @@ class ViewController: UIViewController, UITextFieldDelegate {
         table.translatesAutoresizingMaskIntoConstraints = false
         table.delegate = self
         table.dataSource = self
+        table.register(CustomCell.self, forCellReuseIdentifier: "CustomCell")
```

Comment:
the string you're using for the reuseIdentifier is error prone, store it into a constant value